CoreUtilities/src/au/gov/asd/tac/constellation/utilities/log/ConstellationLogFormatter.java

```diff
@@ -44,9 +47,23 @@ public final String format(final LogRecord record) {
         } else {
             formattedMessage = record.getMessage();
         }
-        sb.append(formattedMessage);
         
-        sb.append(SeparatorConstants.NEWLINE);
+        Throwable error = record.getThrown();
+        if (error != null && error.getMessage() != null && !error.getMessage().isBlank()) {
```

Comment:
the latter 2 conditions can be replaced with `StringUtils.isNotBlank(error.getMessage())` which has a null check included